server/pages/index.js

```diff
@@ -134,7 +134,13 @@ function getDefaultContext( request ) {
 		isFluidWidth: !! config.isEnabled( 'fluid-width' ),
 		abTestHelper: !! config.isEnabled( 'dev/test-helper' ),
 		devDocsURL: '/devdocs',
-		store: createReduxStore()
+		store: createReduxStore(),
+		css: new Set(),
+		insertCss: ( ...styles ) => {
+			styles.forEach( function( style ) {
+				context.css.add( style._getCss() )
```

Comment:
I guess because the same component can be mounted several times and we don't want to have too many duplications of our css styles.